### PR TITLE
Possibly optimize auto-split a bit

### PR DESCRIPTION
**Pull request by joeyballentine**
I did this when messing around with the memory estimation stuff

### CODE REVIEW

File: backend/src/nodes/utils/pytorch_auto_split.py [hunk 1]
```diff
@@ -53,7 +53,6 @@ def auto_split_process(
         d_img = None
         try:
             device = torch.device(os.environ["device"])
-            model = model.to(device)
```
Comment (RunDevelopment) [inline backend/src/nodes/utils/pytorch_auto_split.py:56]: Why did you remove this?

Comment (joeyballentine) [reply backend/src/nodes/utils/pytorch_auto_split.py:56]: Because it's already on the GPU when it loads the model in. this is unnecessary. At best it's a no-op, at worst it just wastes time. It never moves it off the device, so doing it once is all it needs.

**Review by RunDevelopment — APPROVED** [review]
